airbyte-cdk/python/unit_tests/sources/streams/http/test_http.py

```diff
@@ -509,3 +509,18 @@ def test_default_get_error_display_message_handles_http_error(mocker):
 
     http_err_msg = stream.get_error_display_message(requests.HTTPError())
     assert http_err_msg == "my custom message"
+
+@pytest.mark.parametrize(
+    "test_name, base_url, path, expected_full_url",[
+        ("test_no_slashes", "https://airbyte.io", "my_endpoint", "https://airbyte.io/my_endpoint"),
+        ("test_trailing_slash_on_base_url", "https://airbyte.io/", "my_endpoint", "https://airbyte.io/my_endpoint"),
+        ("test_trailing_slash_on_base_url_and_leading_slash_on_path", "https://airbyte.io/", "/my_endpoint", "https://airbyte.io/my_endpoint"),
+        ("test_leading_slash_on_path", "https://airbyte.io", "/my_endpoint", "https://airbyte.io/my_endpoint"),
+        ("test_trailing_slash_on_path", "https://airbyte.io", "/my_endpoint/", "https://airbyte.io/my_endpoint/"),
+        ("test_nested_path_no_leading_slash", "https://airbyte.io", "v1/my_endpoint", "https://airbyte.io/v1/my_endpoint"),
+        ("test_nested_path_with_leading_slash", "https://airbyte.io", "/v1/my_endpoint", "https://airbyte.io/v1/my_endpoint"),
+    ]
+)
+def test_join_url(test_name, base_url, path, expected_full_url):
```

Comment:
@sherifnada I added some tests to the HttpStream to confirm there are no duplicate `/`. Does this cover your concerns?

Reply:
yes! wonderful. Thanks!